sdk/core/core-rest-pipeline/review/core-rest-pipeline.api.md

```diff
@@ -371,7 +376,7 @@ export interface RedirectPolicyOptions {
 }
 
 // @public
-export type RequestBodyType = NodeJS.ReadableStream | (() => NodeJS.ReadableStream) | ReadableStream<Uint8Array> | (() => ReadableStream<Uint8Array>) | Blob | ArrayBuffer | ArrayBufferView | FormData | string | null;
+export type RequestBodyType = NodeJSReadableStream | (() => NodeJSReadableStream) | ReadableStream<Uint8Array> | (() => ReadableStream<Uint8Array>) | Blob | ArrayBuffer | ArrayBufferView | FormData | string | null;
```

Comment:
This change could be problematic for the input case since at least from TypeScript's point of view, not every `NodeJS.ReadableStream` will satisfy the new contract. I wouldn't be surprised if we have references to `NodeJS.ReadableStream` in client libraries that would also need to be updated if we made this change.

Reply:
Thanks @timovv, I addressed it.